Description: disable tests that fail when run with --all-targets.
Author: Peter Michael Green <plugwash@debian.org>
Bug-Debian: https://bugs.debian.org/1005050
---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2022-05-01

--- rust-csv-1.1.6.orig/tests/tests.rs
+++ rust-csv-1.1.6/tests/tests.rs
@@ -21,7 +21,7 @@ static SMALLPOP_COLON: &'static str =
 static SMALLPOP_NO_HEADERS: &'static str =
     include_str!("../examples/data/smallpop-no-headers.csv");
 
-#[test]
+/*#[test]
 fn cookbook_read_basic() {
     let mut cmd = cmd_for_example("cookbook-read-basic");
     let out = cmd_output_with(&mut cmd, SMALLPOP.as_bytes());
@@ -338,7 +338,7 @@ fn tutorial_perf_core_01() {
     let mut cmd = cmd_for_example("tutorial-perf-core-01");
     let out = cmd_output_with(&mut cmd, WORLDPOP.as_bytes());
     assert_eq!(out.stdout(), "11\n");
-}
+}*/
 
 #[test]
 fn no_infinite_loop_on_io_errors() {
